src/TestRunner.java

```diff
@@ -0,0 +1,61 @@
+import java.lang.annotation.*;
+import java.lang.reflect.Method;
+import java.util.*;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+@interface Test {
+    int priority() default 5;
+}
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+@interface BeforeSuite {}
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+@interface AfterSuite {}
+
+public class TestRunner {
```

Comment:
I haven't found any code that makes your test suite run by some priorities. Create annotation `@Order`, specify a priority inside `@Order` and add some logic to make your test methods run by priority.